backend/service/service_tokenstats.go

```diff
@@ -0,0 +1,169 @@
+package service
+
+import (
+	"github.com/irisnet/explorer/backend/model"
+	"github.com/irisnet/explorer/backend/lcd"
+	"github.com/irisnet/explorer/backend/orm/document"
+	"github.com/irisnet/explorer/backend/logger"
+	"github.com/irisnet/explorer/backend/utils"
+	"fmt"
+	"sync"
+)
+
+type TokenStatsService struct {
+	BaseService
+}
+
+func (service *TokenStatsService) QueryTokenStats() (model.TokenStatsVo, error) {
+
+	var (
+		tokenStats  model.TokenStatsVo
+		supply      lcd.Coin
+		circulation lcd.Coin
+	)
+
+	banktokenstats, err := lcd.GetBankTokenStats()
```

Comment:
this http request why not use a goroutine

Reply:
I think no necessary do it, because GetBankTokenStats request costs time so few. that's sure to use a goroutine?

Reply:
have done